endpoints/openrtb2/amp_auction.go

```diff
@@ -162,17 +163,28 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 			}
 		}
 	}
+	// Extract any errors
+	var extResponse openrtb_ext.ExtBidResponse
+	var respErrors map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderError
```

Comment:
you can simplify this by removing `respErrors`. Just replace `Errors:    respErrors,` with `Errors:    extResponse.Errors`,